gr-blocks/lib/moving_average_XX_impl.h.t

```diff
@@ -25,6 +25,8 @@
 #ifndef @GUARD_NAME_IMPL@
 #define @GUARD_NAME_IMPL@
 
+#include <vector>
```

Comment:
I don't think we have a standard for this, but it seems like the C++ world in general is standardizing on including standard headers last, not first. This is to avoid the case where you accidentally hide a missing include by pre-including stuff early.

Reply:
I would also not consider this a blocking issue.